backpack/extensions/firstorder/gradient/base.py

```diff
@@ -1,17 +1,67 @@
+"""Calculates the gradient."""
 from backpack.extensions.firstorder.base import FirstOrderModuleExtension
 
 
 class GradBaseModule(FirstOrderModuleExtension):
-    def __init__(self, derivatives, params=None):
+    """Calculates the gradient.
+
+    This class implements functions with method names from params.
+
+    Passes the calls for the parameters to the derivatives class.
+    Implements functions with method names from params.
+
+    If child class wants to overwrite these methods
+    - for example to support an additional external module -
+    it can do so using the interface for parameter "param1"
+
+    param1(ext, module, g_inp, g_out, bpQuantities):
+        return batch_grads
+
+    In this case, the method is not overwritten by this class.
+    """
+
+    def __init__(self, derivatives, params):
+        """Initializes all methods.
+
+        If the param method has already been defined, it is left unchanged.
+
+        Args:
+            derivatives(backpack.core.derivatives.basederivatives.BaseParameterDerivatives):# noqa
```

Comment:
Can you specify the exact error code that should be ignored here by the docstring checker? [Here](https://github.com/f-dangel/cockpit/blob/9ea1dd4fe60d43f4cec0220f6e5f19e4ae81ed9e/cockpit/quantities/hooks/base.py#L25) is an example. Otherwise, other potential errors will be ignored, too.